invenio_webhooks/views.py

```diff
@@ -146,7 +152,12 @@ def _get_event(receiver_id, event_id):
             receiver_id=receiver_id, id=event_id
         ).first_or_404()
 
-        if event.user_id != request.oauth.access_token.user_id:
+        try:
+            user_id = request.oauth.access_token.user_id
+        except AttributeError:
+            user_id = current_user.get_id()
```

Comment:
`request.oauth.access_token.user_id` seems to return and integer, as does `current_user.id`, but `current_user.get_id()` returns a string.
Do we want/need integers or strings here?